### ISSUE TITLE
Add support for joi-browser.

### ISSUE THREAD

**Issue by Veetaha** [hjkcai/typesafe-joi#15]
I am sorry, if there indeed exists a clever way to use `joi-browser` when importing this library, but I failed to find it. May ask you to add a file like `typesafe-joi/browser` that would export `joi-browser` instead of plain `joi` and use the same typings?

**Comment by hjkcai** [comment]
Yes, but joi-browser is out of date. It may not match the behavior of the current joi. @jeffbski

I prefer creating a new package because the official joi does not have browser version at all. So typesafe-joi should not have neither.

**Comment by hjkcai** [comment]
@jeffbski Could you please test if joi-browser is compatible with the latest joi? I would like to submit a PR to integrate typesafe-joi into joi-browser.
